resources/ext/ezbake.conf

```diff
@@ -14,6 +29,10 @@ ezbake: {
              }
 
       debian: { dependencies: ["puppet-agent"],
+               # see redhat comments on why this is terrible
```

Comment:
Same comments here as for the redhat foss section - not sure we want any of this.  The rundir is different for Puppet v4 and I think this will get reworked for SERVER-414 anyway.